ibm/service/power/resource_ibm_pi_cloud_connection_network_attach.go

```diff
@@ -13,13 +13,12 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 
 	st "github.com/IBM-Cloud/power-go-client/clients/instance"
-	"github.com/IBM-Cloud/power-go-client/helpers"
 	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
 	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
 )
 
 const (
-	PICloudConnectionNetworkId = "pi_network_id"
```

Comment:
According to the documentation:
https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/pi_cloud_connection_network_attach

cloud instance, cloud connection and cloud connection network id are all arguments.
Please relabel as Arg_.

Also this definition to ibm_pi_constants.go.